Description: Disable armhf gdb tests
 Disables armhf gdb tests due to GDB bugs
Author: Zixing Liu <zixing.liu@canonical.com>
Forwarded: not-needed
Last-Update: 2023-11-09
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: rustc/src/tools/compiletest/src/runtest.rs
===================================================================
--- rustc.orig/src/tools/compiletest/src/runtest.rs
+++ rustc/src/tools/compiletest/src/runtest.rs
@@ -756,6 +756,7 @@ impl<'test> TestCx<'test> {
 
         let test_cx = TestCx { config: &config, ..*self };
 
+        #[cfg(disable_debuginfo)]
         test_cx.run_debuginfo_gdb_test_no_opt();
     }
 
